website/webpack.config.js

```diff
@@ -127,11 +125,39 @@ const addProdConfig = config => {
     })
   );
 
+  config.externals = {
+    'highlight.js': 'hljs',
+    'h3-js': 'h3',
+    'deck.gl': 'deck',
+    '@deck.gl/aggregation-layers': 'deck',
+    '@deck.gl/core': 'deck',
+    '@deck.gl/extensions': 'deck',
+    '@deck.gl/geo-layers': 'deck',
+    '@deck.gl/layers': 'deck',
+    '@deck.gl/mesh-layers': 'deck',
+    '@loaders.gl/core': 'loaders',
+    '@luma.gl/core': 'luma',
+    'mapbox-gl': 'mapboxgl'
+  };
+
   return Object.assign(config, {
     mode: 'production'
   });
 };
 
+// s2-geometry cannot be directly included in a script tag
+const S2_CONFIG = {
```

Comment:
I have come to find out that the script tag usage as documented in https://github.com/uber/deck.gl/blob/641f5b1521754dbc03917f9a8078badc5c7ef248/docs/layers/s2-layer.md#installation does not work. The s2-geometry lib contains a `require` so it is not umd compliant.

@ibgreen I really don't want to bundle it with the main module. Any suggestions? Shall we distribute a pre-bundled version of s2-geometry with geo-layers?

Reply:
Switching to use https://bundle.run